src/OrchardCore.Modules/Orchard.Liquid/Filters/TimeZoneFilter.cs

```diff
@@ -59,7 +59,7 @@ public TimeZoneFilter(ISiteService siteService, IClock clock)
 
             _localTimeZone = _localTimeZone ?? TimeZoneInfo.FindSystemTimeZoneById((await _siteService.GetSiteSettingsAsync()).TimeZone);
 
-            var local = TimeZoneInfo.ConvertTime(value, _localTimeZone);
+            var local = TimeZoneInfo.ConvertTime(value, _localTimeZone).LocalDateTime;
```

Comment:
Ah ok, i did this because a binder was failing somewhere because the type was a `DateTimeOffset`.
Nedd to think about this.